.github/workflows/chart-release.yml

```diff
@@ -1,48 +1,25 @@
-name: Release Charts
+name: Deploy Helm Charts to GitHub Pages
 
 on:
   push:
-    paths:
-      - 'charts/**'
     branches:
       - main
 
-permissions:
-  id-token: write
-  contents: read
-
 jobs:
-  release:
+  deploy:
     runs-on: ubuntu-latest
+
     steps:
-      - name: Turnstyle
-        uses: softprops/turnstyle@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Checkout
-        uses: actions/checkout@v2
-      - uses: azure/setup-helm@v1
-      - name: Login Azure
-        uses: azure/login@v1
-        with:
-          client-id: ${{ secrets.AZURE_CLIENT_ID }}
-          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      - name: Checkout repository
+        uses: actions/checkout@v4
 
-      - name: Reindex
+      - name: Update Helm index
```

Comment:
Fix the comment: https://github.com/open-cluster-management-io/helm-charts/pull/11#discussion_r1623888512